vault/service/init.sls

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls syntax=yaml softtabstop=2 tabstop=2 shiftwidth=2 expandtab autoindent
+
+{% from "vault/map.jinja" import vault with context %}
+
+vault-service-init-file-managed:
+  file.managed:
+    - name: {{ vault.service.path }}
+    - source: {{ vault.service.source }}
+    - template: jinja
+    - watch_in:
+      - service: vault
```

Comment:
Refer to [this comment](https://github.com/saltstack-formulas/vault-formula/pull/23/files?file-filters%5B%5D=.sls&file-filters%5B%5D=.yaml&hide-deleted-files=true#r268416313).